templates/etc/td-agent/td-agent.conf

```diff
@@ -28,10 +28,17 @@
   buffer_chunk_limit 1M
   # Flush logs every 5 seconds, even if the buffer is not full.
   flush_interval 5s
-  # Never wait longer than 5 minutes between retries.
+  # Enforce some limit on the number of retries.
+  disable_retry_limit false
+  # The maximum number of retries for sending a chunk. After this many of
+  # retries, the given chunk will be discarded.
+  retry_limit 8
+  # Wait 1 second before the first retry. The wait interval will be doubled on
+  # each following retry (2s, 4s, 8s...) until it reaches the max_retry_wait.
```

Comment:
It could be good to add that the retry time is randomised: https://docs.fluentd.org/v1.0/articles/buffer-plugin-overview#how-exponential-backoff-works

Reply:
Good point! Added some comments to point that out.

Reply:
Careful -- we're still on fluentd 0.12. The fluentd 1.0 retry controls are very different. Once we are done with 0.12, we can update to the newer ones, and all of the names will change.

Reply:
Right. Just removed the comment.